.github/workflows/load-tests.yml

```diff
@@ -0,0 +1,182 @@
+name: Load Tests
+
+on: [pull_request]
```

Comment:
we need to be sure that this action is running only on main and development branch ( on push )